### ISSUE TITLE
Conflict with YouTube playlist dropdown

### ISSUE THREAD

**Issue by kenorb** [philc/vimium#1882]
When I'm trying to add some [video](https://www.youtube.com/watch?v=8ZPqFVM6j6A) to the playlist, and I type 'free', it doesn't let me, as it's refreshing the page (probably on `r` keystroke).

I'm using: Version 1.53

**Comment by smblott-github** [comment]
Ouch!
Confirmed.

**Comment by smblott-github** [comment]
Here's what's happening...

Whenever the typed keys match existing playlists, youtube refreshes the list of playlists and **blurs the input** (so Vimium drops out of insert mode, as it should).  Youtube is then catching keys elsewhere, and inserting them into the input.  Depending upon the characters typed and the matching playlists, the input may or may not be focused, so Vimium may or may not be in insert mode.

Because Vimium isn't in insert mode, keys like `r` (or worse still `x`!) are handled by Vimium, with undesirable effects.

Workaround?  I can't think of anything better than disabling Vimium on youtube.

**Comment by jellytux** [comment]
What needs to be disabled for the most part is not Vimium at all, but Youtube hotkeys themselves. Some Vimium plugin would be nice.

**Comment by smblott-github** [comment]
Thanks, @jellytux.  However, it's not actually YoutTube hotkeys that are causing the problem.  The user is typing text into what looks like an input, which is kind of fundamental to the page.  Except, sometimes it is a real input, and sometimes it's not.  It's just the weird way YouTube have implemented their interface.

**Comment by jellytux** [comment]
@smblott-github I don't think this is the only issue though, video player hotkeys also conflict. You could just disable Vimium if any inputs are focused, that shouldn't be too hard to implement.
